packages/dataviews/src/filter-summary.tsx

```diff
@@ -188,8 +243,8 @@ export default function FilterSummary( {
 	addFilterRef,
 	openedFilter,
 	...commonProps
-} ) {
-	const toggleRef = useRef();
+}: FilterSummaryProps ) {
+	const toggleRef = useRef< HTMLDivElement | null >( null );
```

Comment:
React uses a little trick here to differentiate between refs that we update manually and refs that the system manages. This seems like a ref the system manages. If we make this change:

```suggestion
	const toggleRef = useRef< HTMLDivElement >( null );
```

React recognizes that it's a `RefObject` instead of a `MutableRefObject`. This basically allows assignment to `current` or not:

```ts
// Ok
const toggleRef = useRef< HTMLDivElement | null >( null );
toggleRef.current = document.createElement( 'div' );

// Not OK (not mutable)
const toggleRef = useRef< HTMLDivElement >( null );
toggleRef.current = document.createElement( 'div' ); // type error
```